pkg/cluster/nodes/create.go

```diff
@@ -130,6 +131,15 @@ func createNode(name, image, clusterLabel string, role config.NodeRole, extraArg
 		"--entrypoint=/usr/local/bin/entrypoint",
 	}
 
+	// pass proxy environment variables to be used by node's docker deamon
+	if os.Getenv("HTTP_PROXY")!= "" {
+	  runArgs = append(runArgs,"-e HTTP_PROXY=",os.Getenv("HTTP_PROXY"))
+	}
+
+	if os.Getenv("HTTPS_PROXY") != "" {
+	  runArgs = append(runArgs,"-e HTTPS_PROXY=",os.Getenv("HTTPS_PROXY"))
+	}
```

Comment:
please run `./hack/update-gofmt.sh` and include the changes to be able to pass CI.
if you are using editors like VS Code it should format the code correctly for you.

non-blocking nit would be to store the ENV variables into intermediate variables - e.g.:
```
http_proxy := getenv(...)
if http_proxy != "" {
    ....
}
https_proxy := getenv(...)
if https_proxy != "" {
    ....
}
```